olm/addDigests.sh

```diff
@@ -124,10 +124,22 @@ yq -riY "( .spec.relatedImages ) += [${RELATED_IMAGES}]" ${CSV_FILE}
 yq -riY "( .spec.install.spec.deployments[0].spec.template.spec.containers[0].env ) += [${RELATED_IMAGES_ENV}]" ${CSV_FILE}
 sed -i "${CSV_FILE}" -r -e "s|tag: |# tag: |"
 echo -e "$(cat ${SCRIPTS_DIR}/license.txt)\n$(cat ${CSV_FILE})" > ${CSV_FILE}
-
 echo "[INFO] CSV updated: ${CSV_FILE}"
 
 if [[ ${OPERATOR_FILE} ]]; then
+  # delete previous `RELATED_IMAGES`
+  envVarLength=$(cat "${OPERATOR_FILE}" | yq -r ".spec.template.spec.containers[0].env | length")
+  i=0
+  while [ "${i}" -lt "${envVarLength}" ]; do
+    envVarName=$(cat "${OPERATOR_FILE}" | yq -r '.spec.template.spec.containers[0].env['${i}'].name')
+    if [[ ${envVarName} =~ plugin_registry_image ]] || [[ ${envVarName} =~ devfile_registry_image ]]; then
```

Comment:
If you preserve only two env vars, does it make sense recreate whole list with these two?

Reply:
That's a regexp

Reply:
Will it ever preserve more than 2 env vars?

Reply:
Sure

Reply:
I see, due to some reason I missed the negations, sorry.